Google.Api.Gax.Grpc/Rest/RestGrpcAdapter.cs

```diff
@@ -74,5 +75,14 @@ public sealed class RestGrpcAdapter : GrpcAdapter
                 // Anything else (including all 1xx and 3xx) maps to Unknown
                 _ => StatusCode.Unknown
             };
+
+        private class GrpcChannelFactory
+        {
+            private RestServiceCollection _serviceCollection;
+
+            internal GrpcChannelFactory(RestServiceCollection serviceCollection) =>
+                _serviceCollection = serviceCollection;
+
+        }
```

Comment:
Left over maybe?

Reply:
Yes - was expecting to need more, but given that it's *just* a service collection, I figured I could put that into the dictionary directly.